Jenkinsfile

```diff
@@ -98,6 +115,26 @@ pipeline {
         } }
       }
     }
+    stage('Smoke Test') {
+      when { allOf {
+        expression { return !PIPELINE_CONTROL.ci_skip }
+      } }
+      steps {
+        timeout(time: 10, unit: 'MINUTES') { script {
+          def vscodePackageJson = readJSON file: "package.json"
+          def version = "v${vscodePackageJson.version}"
+          sh "git tag ${version}"
```

Comment:
Is the intention to push a snapshot git-tag to GH?
If not, I believe this can be removed from the stage.

Reply:
Should remove these...